docs/preview/02-getting-started.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_label: Getting started
+---
+
+# Getting started with Arcus Messaging
+**Welcome to Arcus Messaging!** 🎉
+
+This page is dedicated to be used as a walkthrough on how to set up Arcus Messaging in a new or existing project. Arcus Messaging is an umbrella term for a set of NuGet packages that kickstart your messaging solution.
+
+**Used terms:**
+* *message handler:* a custom implementation of an Arcus Messaging-provided interface that processes a deserialized Azure Service bus message.
+* *message pump:* an Arcus Messaging-provided registered service that receives Azure Service bus messages for you, and "pump's" them through your *message handlers*.
+
+## The basics
+> 👉 Arcus Messaging is currently only supported for [Azure Service bus](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-messaging-overview) solutions.
```

Comment:
```suggestion
> 👉 Arcus Messaging is currently only supported for [Azure Service Bus](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-messaging-overview) solutions.
```

Reply:
Noticed that Microsoft sometimes uses this format: `Azure Component name` like `Azure Key vault`. Shouldn't we follow that?

Reply:
I was looking at things like this:
![image](https://github.com/user-attachments/assets/b14361d0-3c93-4d54-8fb3-219c5268892d)

![image](https://github.com/user-attachments/assets/d2131984-7e0d-446d-9456-d6382d03a01b)

Reply:
Ok, let me check in other places as well.

Reply:
Done in https://github.com/arcus-azure/arcus.messaging/pull/546/commits/b74cc4cc0f5fff6065e845bead14de5365ae1b02